src/coreclr/tools/aot/ILCompiler.Diagnostics/PerfMapWriter.cs

```diff
@@ -108,6 +108,7 @@ internal record struct PerfmapTokensForTarget(PerfMapOSToken OperatingSystem, Pe
 
         private static PerfmapTokensForTarget TranslateTargetDetailsToPerfmapConstants(TargetDetails details)
         {
+            // TODO: What about ios and iossimulator
```

Comment:
@MichalStrehovsky should we just extend the `PerfMapOSToken` enumeration for now ?

Reply:
As long as we're not bringing up CoreCLR-JIT on iOS, this doesn't matter. It's used for CoreCLR ReadyToRun only.

Reply:
Understood, I removed the comment.